app/templates/application.hbs

```diff
@@ -2,13 +2,22 @@
 
 <AuApp>
   <AppHeader />
-  <NavigationBar />
-
+  {{#if this.isIndex}}
```

Comment:
Is there a more Ember-esque solution for this? Whether it be an outlet/yield/block situation or a variable that can easily be set in the route itself. The current structure  - whilst clean - does cause a bit of a unclarity

Reply:
Do you have an example of how this works? Or do I simply create two components that wrap the page content and set those on each template?

Reply:
Yea, that's indeed what I meant. But I don't see it as a blocker here.